environment.yml

```diff
@@ -21,3 +21,6 @@ dependencies:
   - pp-sketchlib >=1.5.1
   - graph-tool
   - requests
+  - flask 
+  - flask-cors
+  - networkx
```

Comment:
Can you also add these to the requirements and mock imports in the `docs/` subdirectory?